lesson_08/homework_08.py

```diff
@@ -0,0 +1,27 @@
+#Вирішення 1
+number_list = ["1,2,3,4",
+    "1,2,3,4,50",
+    "qwerty1,2,3"]
+def find_sum_numbers(some_list):
+    for element in some_list:
+        try:
+            result = 0
+            for i in element.split(','):
+                result = result + int(i)
+            print(result)
```

Comment:
```suggestion
            print(result)
```
помилка в тому, що як я не раз казав функція повинна щось ПОВЕРТАТИ. А у вас поверне None
Коротше підказка така: якщо це не функція для виводу або вводу -то в ній прінтів бути не повинно.

Reply:
Додала 3 рішення в новий пул реквест https://github.com/alex-pancho/aqa_py_290424/pull/67